src/pages/index.js

```diff
@@ -80,12 +228,17 @@ addCardButton.addEventListener("click", () => {
 
 editProfileButton.addEventListener("click", function inputProfileInfo() {
   profFormPopup.open();
-  const currUserInfo = profUserInfo.getUserInfo();
-  profNameInput.value = currUserInfo.name;
-  profDescrInput.value = currUserInfo.description;
+  profNameInput.value = profName.textContent;
+  profDescrInput.value = profDescr.textContent;
   profileFormValidator.resetValidation();
 });
 
-// VALIDATION
+editAviButton.addEventListener("click", () => {
+  editAviFormPopup.open();
+  editAviFormValidator.resetValidation();
```

Comment:
If you don't reset the form on opening, then you don't need to remove errors from the inputs

`resetValidation` is not needed here


You reset the form after submitting so, you need to disable the button also in `then` after resetting the form there